Roles/Core/CustomRoleManager.cs

```diff
@@ -414,7 +410,8 @@ public static string GetMarkOthers(PlayerControl seer, PlayerControl seen, bool
     /// </summary>
     public static string GetLowerTextOthers(PlayerControl seer, PlayerControl seen, bool isForMeeting = false, bool isForHud = false)
     {
-        if (!LowerOthers.Any()) return string.Empty;
+        var LowerOthers = new HashSet<Func<PlayerControl, PlayerControl, bool, bool, string>>();
+        LowerOthers = AllEnabledRoles.Select(lower => (Func<PlayerControl, PlayerControl, bool, bool, string>)lower.GetLowerTextOthers).FilterDuplicates();
```

Comment:
idk, maybe the old method is better, because this code will check the list every second for duplicates and Idk if this will affect performance

Reply:
alright I'll make it added on start